README.md

```diff
@@ -356,15 +356,15 @@ The STH component provides the user with 2 mechanisms to configure the component
 
 * Environment variables, which can be set assigning values to them or using the `sth_default.conf` file if a packaged
 version of the STH component is used.
-* The [`config.js`](https://github.com/telefonicaid/IoT-STH/blob/develop/config.js) file located at the root of the STH component code, a JSON formatted file including the configuration properties.
+* The `config.js` file located at the root of the STH component code, a JSON formatted file including the configuration properties.
```

Comment:
Uhm, since I was asked to keep both names (STH and Comet) and until the STH is officially presented in some FIWARE conference, I would leave it as STH... :) Are you OK with this? :)